e2e/upgrade-rbd.go

```diff
@@ -149,13 +167,58 @@ var _ = Describe("RBD Upgrade Testing", func() {
 				if err != nil {
 					e2elog.Failf("failed to load application with error %v", err)
 				}
+				label := map[string]string{
+					"app": "upgrade-testing",
+				}
 				app.Namespace = f.UniqueName
-				app.Labels = map[string]string{"app": "upgrade-testing"}
+				app.Labels = label
 				pvc.Spec.Resources.Requests[v1.ResourceStorage] = resource.MustParse(pvcSize)
 				err = createPVCAndApp("", f, pvc, app, deployTimeout)
 				if err != nil {
 					e2elog.Failf("failed to create pvc with error %v", err)
 				}
+				opt := metav1.ListOptions{
+					LabelSelector: "app=upgrade-testing",
```

Comment:
derive this from the label defined at line 170 so that if we change the label at 170 everything should work

Reply:
Yup, that would be definitely better :+1:

Reply:
But, Since we might need to use the same label value in the later stages, it might introduce some global vars.... Will that be fine?

Reply:
App specific should be fine